dashboard/config/upstream-kcsan.config

```diff
@@ -1,28 +1,33 @@
 #
-# Automatically generated file; DO NOT EDIT.
-# Linux/x86 5.3.0-rc1 Kernel Configuration
+# Linux/x86 5.3.0 Kernel Configuration
 #
 
-# The following configs are added manually, preserve them.
-# CONFIG_DEBUG_MEMORY was once added to mm tree and cause disabling of KASAN,
-# which in turn caused storm of assorted crashes after silent memory
-# corruptions. The config was reverted, but we keep it here for the case
-# it is reintroduced to kernel again.
-CONFIG_DEBUG_MEMORY=y
 # This config can be used to enable any additional temporal debugging
 # features in linux-next tree.
 CONFIG_DEBUG_AID_FOR_SYZBOT=y
 
+# KCSAN enable
+CONFIG_KCSAN=y
+CONFIG_KCSAN_EARLY_ENABLE=n
```

Comment:
Is `=n` even a thing? The standard way to disable something is `is not set`. make olddefconfig may fail to parse `=n`.